ClosedXML.Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -569,6 +569,20 @@ public void TryGetValue_Nullable()
             Assert.IsFalse(ws.Cell("A4").TryGetValue(out double? _));
         }
 
+        [Test]
+        public void ValueSetDateTimeOffset()
+        {
+            IXLWorksheet ws = new XLWorkbook().Worksheets.Add("Sheet1");
+            IXLCell cell = ws.Cell(1, 1);
+            var expected = new DateTimeOffset(2019, 11, 5, 11, 30, 5, new TimeSpan(0));
```

Comment:
Better rename the variable to `input` and make `expected` a string variable with plain string, without using the `format`. Also, we should probably use non-zero offset here.

Reply:
Refactored this.